### PR TITLE
Improve scrolling in video workspace

### PR DESCRIPTION
**Pull request by eldiegod**
Fixes: #1148

### CODE REVIEW

File: src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx [hunk 1]
```diff
@@ -413,219 +417,225 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
...
-              'title',
-              textFieldValidation({ name: 'Video Title', minLength: 3, maxLength: 40, required: true })
+      <FormScrolling>
+        <FormWrapper>
+          <Controller
+            name="assets"
+            control={control}
+            render={() => (
+              <div ref={fileSelectContainerRef}>
+                <MultiFileSelectWrapper width={fileSelectContainerBounds.width}>
```
Comment (kdembler) [inline src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx]: Why is this needed?

Comment (eldiegod) [reply src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx]: it's so the multi file selector sizes itself propperly as it now has to have position: fixed we need to defined it's width manually

Comment (kdembler) [reply src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx]: I've missed that it's `fixed` before. Understandable then, but does it really need to be `fixed`? I did a quick experiment with `position: sticky; top: 32px` and it seems to be working fine without any manual width setting. Can you try that?

**Comment by kdembler** [comment]
One additional thing I think we should fix while we're at it, is the inputs container kind of spanning outside of visible area. You can see that in the video:

https://user-images.githubusercontent.com/12646744/129569747-6d9a041c-09d5-4b89-92de-aa015b45c227.mov

The scrollbar goes under the action bar, which is not really intuitive. Instead it should end when the action bar begins.

File: src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx (hunk 1 above)
Comment (eldiegod) [reply src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx]: sticky worked as well 👍

**Review by kdembler — APPROVED** [review]
